index.html

```diff
@@ -241,6 +241,72 @@ <h2>Skills</h2>
           </div>
         </div>
       </section>
+      <!-- skills-Section-End -->
     </main>
+
+    <footer>
+      <h2 class="title">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+
+      <form action="https://formspree.io/f/xknyzaaw" method="post">
+        <ul>
+          <li>
+            <input
+              id="name"
+              name="user-name"
+              type="text"
+              maxlength="30"
+              placeholder="Full name"
+              required
+            />
+          </li>
+          <li>
+            <input
+              type="email"
+              id="email"
+              name="user-email"
+              placeholder="Email address"
+              required
+            />
+          </li>
+          <li>
+            <textarea
+              name="user-message"
+              id="msg"
+              maxlength="500"
+              placeholder="Enter text here"
+              required
+            ></textarea>
+          </li>
+          <li class="submit-button">
+            <button type="submit">Get in touch</button>
+          </li>
```

Comment:
- [ ] Kindly change your button's interaction effects (hover, pressed, ...etc) to match the Figma design. (See image below)

![image](https://user-images.githubusercontent.com/62290545/159709157-0ae5eabf-2ca6-441c-a3d6-143e7c3733b9.png)